packages/grafana-ui/src/components/JSONFormatter/json_explorer/helpers.ts

```diff
@@ -5,7 +5,7 @@
  * Escapes `"` characters from string
  */
 function escapeString(str: string): string {
-  return str.replace('"', '"');
+  return str.replace(/\\/g, '\\\\').replace(/"/g, '\\"');
```

Comment:
**Question:** couldn't we utilize the `escapeStringForRegex()` in [`grafana-data/src/text/string.ts`](https://github.com/grafana/grafana/blob/431f454d57512813e153bb57e14344345c0e084d/packages/grafana-data/src/text/string.ts#L3-L4)?

It would need to be extended a bit, but maybe better if we have a single utility function for escaping strings?